governing-documents/security.md

```diff
@@ -267,15 +267,20 @@ to understand how to monitor the security advisories published by the project.
 
 ## Private Patch Deployment Infrastructure
 
-Patches to fix Hyperledger project security vulnerabilities are typically
+Patches to fix LF Decentralized Trust project security vulnerabilities are typically
 developed without public visibility by using the private development features of
 GitHub. Projects with maintainers that are not familiar with these capabilities
-are encouraged to contact the [Hyperledger Community Architects] to learn more.
+are encouraged to contact the [LF Decentralized Trust Community Architects] to learn more.
 
-[Hyperledger Community Architects]: community-architects@hyperledger.org
+[LF Decentralized Trust Community Architects]: community-architects@lfdecentralizedtrust.org
 
 **Alternative:** Projects **MAY** document in their security policy document
 that they do use a service other than GitHub for private patch deployment
 infrastructure, or that they don't use any private patch deployment
 infrastructure at all. In either case, the document **MUST** include the details
 of what is used instead.
+
+# FIXMEs
+The following items must be set up in order for this policy to work:
+1. security@lists.lfdecentralizedtrust.org email
+2. community-architects@lfdecentralizedtrust.org email
```

Comment:
I see these email addresses being updated in many other files. Should this section be copied to those other files too? I assume this exists only to serve as a reminder to setup the mailing lists.